libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTrackerNosara.java

```diff
@@ -375,10 +375,14 @@ public static String getEventNameForStat(AnalyticsTracker.Stat stat) {
                 return "reader_article_comment_unliked";
             case READER_ARTICLE_LIKED:
                 return "reader_article_liked";
+            case READER_ARTICLE_LIKED_DETAIL:
+                return "reader_article_liked_detail";
             case READER_ARTICLE_OPENED:
                 return "reader_article_opened";
             case READER_ARTICLE_UNLIKED:
                 return "reader_article_unliked";
+            case READER_ARTICLE_UNLIKED_DETAIL:
+                return "reader_article_unliked_detail";
```

Comment:
To be consistent with Tracks naming conventions the "action" part of the analytic should come after the source an context.  So in this example we'd want it to be `reader_article_detail_unliked`.  Same change for the one above 🙏

Reply:
I updated the naming in 043f0f1.